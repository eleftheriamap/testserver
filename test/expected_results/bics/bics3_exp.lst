
test/exp/test/gen/bics/bics3_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	d16729ce 	sub	x14, x14, #0x9ca, lsl #12
   4:	110030e7 	add	w7, w7, #0xc
   8:	6a6779c9 	bics	w9, w14, w7, lsr #30
   c:	8a000000 	and	x0, x0, x0
