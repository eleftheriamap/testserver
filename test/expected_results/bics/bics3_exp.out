Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 000000000000000c
X08    = 0000000000000000
X09    = 00000000ff636000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = ffffffffff636000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 000000000000000c
PSTATE : N---
Non-zero memory:
0x00000000 : 0xd16729ce
0x00000004 : 0x110030e7
0x00000008 : 0x6a6779c9
0x0000000c : 0x8a000000
